Source/IntegrationTests/IntegrationTests.csproj

```diff
@@ -22,7 +22,9 @@
 
     <ItemGroup>
         <ProjectReference Include="..\DafnyDriver\DafnyDriver.csproj" />
-        <ProjectReference Include="..\DafnyRuntime\DafnyRuntime.csproj" />
+        <ProjectReference Include="..\DafnyRuntime\DafnyRuntime.csproj">
+          <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
```

Comment:
I don't know the meaning of "ReferenceOutputAssembly" in these various projects. @robin-aws  can you please let me know if it's the right thing to do? You wanted to change the dependency to DafnyRuntime for a long time.